<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>549</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="search_button">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>110</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>SEARCH</string>
    </property>
   </widget>
   <widget class="QComboBox" name="filterDropdown">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="currentText">
     <string>Filter</string>
    </property>
    <item>
     <property name="text">
      <string>Filter</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Genre</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rating</string>
     </property>
    </item>
   </widget>
   <widget class="QListWidget" name="bookList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>751</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addBookBtn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>390</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Book</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteBookBtn">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>390</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Book</string>
    </property>
   </widget>
   <widget class="QPushButton" name="filterButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>93</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="title_id">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>360</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>TITLE</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="auhtor_id">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>360</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>AUTHOR</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="genre_id">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>360</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>GENRE</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rating_id">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>360</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="placeholderText">
     <string>RATING</string>
    </property>
   </widget>
   <widget class="QLabel" name="bookDetailsLabel_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>BOOK MANAGEMENT APP</string>
    </property>
   </widget>
   <widget class="QLabel" name="statsid">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>420</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="error_id">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>711</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
